File: WindowsFormsApp1C#/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WindowsFormsApp1
{
    public partial class btnResult : Form
    {
        public btnResult()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            int num;
            num = Convert.ToInt32(txtBox.Text);
                if(num %2==0)
                    { lblResult.Text  = num + " is an even number"; }
                else { lblResult.Text  = num + "is an odd number"; }
            txtBox.Text = " ";
            txtBox.Focus ();

        }
    }
}

File: OddorEvenProject/Form1.vb
Public Class Form1
    Private Sub BtnResult_Click(sender As Object, e As EventArgs) Handles btnResult.Click
        Dim num As Integer
        num = Convert.ToInt32(TextBox1.Text)
        If (num Mod 2 = 0) Then
            lblResult.Text = num & " is an even number"
        Else
            lblResult.Text = num & " is an odd number"

        End If
    End Sub
End Class

